src/actions/activeZone.js

```diff
@@ -1,34 +1,34 @@
-import * as ActionTypes from '../constants/ActionTypes';
-import { asyncDNSRecordFetchList } from './zoneDnsRecords';
-import { asyncZoneFetchAnalytics } from './zoneAnalytics';
-import { asyncZoneRailgunFetchAll } from './zoneRailgun';
-import { asyncZoneFetchSettings } from './zoneSettings';
-import { asyncPluginFetchSettings } from './pluginSettings';
```

Comment:
Do we want `singleQuote: false`? In internal repo we're using `singleQuote: true`